scaleway/resource_instance_security_group.go

```diff
@@ -368,7 +368,7 @@ func securityGroupRuleSchema() *schema.Resource {
 			},
 			"port_range": {
 				Type:        schema.TypeString,
-				Computed:    true,
+				Optional:    true,
```

Comment:
should this be conflit with port?

Reply:
ConflictsWith do no work on neasterd schema